docs/self-hosted-appcircle/install-server/podman.md

```diff
@@ -960,19 +964,21 @@ Self-hosted runner section in docs, has all details about runners and their conf
 
 :::caution
 
-By default, self-hosted runner package has pre-configured `ASPNETCORE_BASE_API_URL` for Appcircle-hosted cloud.
+By default, self-hosted runner package has pre-configured `ASPNETCORE_REDIS_STREAM_ENDPOINT` and `ASPNETCORE_BASE_API_URL` for Appcircle-hosted cloud.
 
+- `webeventredis.appcircle.io:6379,ssl=true`
 - `https://api.appcircle.io/build/v1`
 
-:point_up: You need to change its value with your self-hosted Appcircle server's API URL.
+:point_up: You need to change these values with your self-hosted Appcircle server's Redis and API URL.
 
-Assuming our sample scenario explained above, its value should be
+Assuming our sample scenario explained above, these values should be
 
+- `redis.appcircle.spacetech.com:6379,ssl=false`
```

Comment:
This section should also have a sample case for the HTTPS configuration.

Currently, that's only mentioned in macOS VM documents, but users can also follow other runner installation options.

Reply:
Done ✅